app/controllers/idv/personal_key_controller.rb

```diff
@@ -117,5 +122,14 @@ def in_person_enrollment?
       return false unless IdentityConfig.store.in_person_proofing_enabled
       current_user.pending_in_person_enrollment.present?
     end
+
+    def pii_is_missing
+      user_session[:encrypted_profiles].blank?
```

Comment:
TIL that `{}.blank?` is true!

Reply:
Always glad to pass along information :-)